remote-api/src/test/java/org/alfresco/rest/api/impl/RulesImplTest.java

```diff
@@ -394,10 +395,136 @@ public void testSaveRules_createMultipleRules()
         assertThat(actual).isEqualTo(expected);
     }
 
+    @Test
+    public void testDeleteRuleById() {
+        given(nodesMock.validateNode(eq(RULE_SET_ID))).willReturn(ruleSetNodeRef);
+        given(nodesMock.validateNode(eq(RULE_ID))).willReturn(ruleNodeRef);
```

Comment:
FWIW we don't need the `eq` in these two lines.

Reply:
Thanks for spotting that. I'm cleaning up all `eq(...)` uses in that class as there are way too many not needed.